### PR TITLE
Minor update to Docs on Linear solver curvilinear coordinate

### PR DESCRIPTION
**Pull request by cgilet**
support.

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [ ] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [ ] include documentation in the code and/or rst files, if appropriate

### CODE REVIEW

File: Docs/sphinx_documentation/source/LinearSolvers.rst [hunk 1]
```diff
@@ -388,14 +388,22 @@ to the ghost cell center; :cpp:`maxorder = 3` uses the boundary value and the fi
 Curvilinear Coordinates
 =======================
 
-The linear solvers support curvilinear coordinates including 1D
```
Comment (WeiqunZhang) [inline Docs/sphinx_documentation/source/LinearSolvers.rst:391]: The nodal solver supports RZ too, at least partially.  It does require the user to rescale the problem.  But the solver itself also has to do some additional correction. (`git grep is_rz Src/LinearSolvers/MLMG`)

Comment (cgilet) [reply Docs/sphinx_documentation/source/LinearSolvers.rst:391]: This is what I was trying to say in the next paragraph, but obviously, I was not clear. I'll try again...

Comment (cgilet) [reply Docs/sphinx_documentation/source/LinearSolvers.rst:391]: @WeiqunZhang Can you clarify if `setMetricTerm(false)` actually turns off the correction terms for the nodal solver or not. As I read the code, it appears to me as though that doesn't affect the nodal solver, and instead `setRZCorrection(false)` must be used.

Comment (WeiqunZhang) [reply Docs/sphinx_documentation/source/LinearSolvers.rst:391]: That's correct.

**Review by WeiqunZhang — APPROVED** [review]

**Pull request merged**